homeassistant/components/kostal_plenticore/switch.py

```diff
@@ -0,0 +1,183 @@
+﻿"""Platform for Kostal Plenticore switches."""
+from __future__ import annotations
+
+from abc import ABC
+from datetime import timedelta
+import logging
+from typing import Any
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    DOMAIN,
+    SWITCH_SETTINGS_DATA,
+)
+from .helper import (
+    SettingDataUpdateCoordinator,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
+):
+    """Add kostal plenticore Sensors."""
```

Comment:
stale

Reply:
Copied from senor.py to keep things consistent. Why was it ok, when maintainer submitted the code for senor.py?

Reply:
fixed in ca61960